src/MakersOfDenmark.Domain/Models/MakerSpace.cs

```diff
@@ -16,6 +18,18 @@ public class MakerSpace : Entity<Guid>
         public AccessType AccessType { get; set; }
         public Organization Organization { get; set; }
         public ICollection<Tool> Tools { get; set; } = new List<Tool>();
-
+        public ICollection<MODUserFollowMakerSpace> Followers { get; private set; } = new List<MODUserFollowMakerSpace>();
+        public void FollowMakerSpace(MODUser newFollower)
```

Comment:
Synes det ser rigtig fornuftigt ud, men ville det ikke være mere oplagt at lave dette til Request/Response?

Reply:
Jeg er enig med at der skal oprettes et Request/Response til denne use case. Jeg kan dog også se fornuften i at have denne logik i MakerSpace for at sørge for at vi har en fornuftig afgrænsning af, hvad der er domæne logik og hvad der er applikations logik.

I et requesthandleren vil man blot loade MakerSpaced ud med sine Followers eksekverer denne metode på makerspaced. Dette giver I min mening en mere læselig logik i handleren og gør vores domæne mere testbart fordi vi ikke er nød til at Mocke dbcontexten, som vi ville være i en handler.